.drone.yml

```diff
@@ -89,9 +89,7 @@ steps:
       chown -R test:test . &&
       su -c 'ASAN_OPTIONS=detect_leaks=0 ctest --output-on-failure' test"
     # Static analysis with clang-tidy
-    - /bin/bash -c "
-      ln -s build/compile_commands.json . &&
-      run-clang-tidy-6.0"
+    - "! run-clang-tidy-6.0 -p build | grep -A 5 ': error:'"
```

Comment:
If we're trying to reduce the noise we could also use the `-quiet` command line flag I guess. What do you think?

Reply:
Reducing the noise is of course only a side effect of the desired effect to make the CI fail if there are errors. However, since I need to rebase anyway, I can add it.

Reply:
Well it's your grep which make me thing about that. :-)